airflow/cli/commands/db_command.py

```diff
@@ -187,9 +188,21 @@ def shell(args):
 
 
 @cli_utils.action_cli(check_db=False)
-def check(_):
+def check(args):
     """Runs a check command that checks if db is available."""
-    db.check()
+    retries: int = args.retry
```

Comment:
You can also consider using - https://tenacity.readthedocs.io/en/latest/ for retries it gives you many options

Reply:
yeah. using tenacity (especially that we use it, is a good idea here).

Reply:
Thank you for the suggestion. Will work on incorporating `tenacity` right away.